pkg/controller/sync/controller.go

```diff
@@ -605,8 +613,9 @@ func (s *FederationSyncController) syncToClusters(selectedClusters, unselectedCl
 
 // getClusterVersions returns the cluster versions populated in the current
 // propagated version object if it exists.
-func getClusterVersions(adapter federatedtypes.FederatedTypeAdapter, template,
-	override pkgruntime.Object, propagatedVersion *fedv1a1.PropagatedVersion) map[string]string {
+func getClusterVersions(adapter federatedtypes.FederatedTypeAdapter,
```

Comment:
(No action required) In future please separate non-functional changes from functional changes.